src/frontends/lean/json.cpp

```diff
@@ -4,8 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 
 Author: Gabriel Ebner
 */
-#if defined(LEAN_SERVER)
```

Comment:
Yes, the `LEAN_SERVER` ifdef was added to support compilers where compiling the server support did not work. I called the option `LEAN_SERVER` because it disables the server module.